models/archive-artist.php

```diff
@@ -0,0 +1,251 @@
+<?php
+/**
+ *  Copyright (c) 2021. Geniem Oy
+ */
+
+use TMS\Theme\Base\Traits\Pagination;
+use TMS\Theme\Filharmonia\PostType\Artist;
+use TMS\Theme\Filharmonia\Taxonomy\ArtistCategory;
+
+/**
+ * Archive for Artist CPT
+ */
+class ArchiveArtist extends BaseModel {
+
+    use Pagination;
+
+    /**
+     * Search input name.
+     */
+    const SEARCH_QUERY_VAR = 'artist-search';
+
+    /**
+     * Artist category filter name.
+     */
+    const FILTER_QUERY_VAR = 'artist-filter';
+
+    /**
+     * Pagination data.
+     *
+     * @var object
+     */
+    protected object $pagination;
+
+    /**
+     * Hooks
+     */
+    public static function hooks() : void {
+        add_action(
+            'pre_get_posts',
+            [ __CLASS__, 'modify_query' ]
+        );
+    }
+
+    /**
+     * Get search query var value
```

Comment:
Superpieni detalji, mutta meidän lintterit ei taida valittaa, jos kommentit ei pääty pisteeseen (entisessä firmassa valitti, niin iskostunu mulle tavaksi laittaa). Tässä filussa näkyy molempia tapoja käytettävän (pisteellä ja ilman). Muuta, jos haluat ehkä ajatuksena vaan tämmönen. :)

Reply:
Projektissa on tosiaan käytetty molempia tapoja, joten annetaan tällä erää mennä näin :) Mutta olen samaa mieltä siitä että yhdenmukaisuuden nimissä olisi kiva valita tavoista toinen!